sdk/storage/Azure.Storage.Blobs/src/AssemblyInfo.cs

```diff
@@ -3,13 +3,11 @@
 // license information.
 
 using System.Runtime.CompilerServices;
-using Azure.Core;
 
 [assembly: InternalsVisibleTo("Azure.Storage.Blobs.Tests, PublicKey=" +
     "0024000004800000940000000602000000240000525341310004000001000100d15ddcb2968829" +
     "5338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc" +
     "012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265" +
     "e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593d" +
     "aa7b11b4")]
-[assembly: AzureSdkClientLibrary("Storage.Blobs")]
```

Comment:
Does the build automagically add the `AssemblyInformationalVersionAttribute`?  Or should we be adding it here as part of this change?

Reply:
Yep, it does, it's set to package version automatically.

Reply:
Awesome - having one less thing to touch is always great.